notebook/static/notebook/less/variables.less

```diff
@@ -26,3 +26,6 @@
 @cell_padding:                  6px;
 @cell_border_width:             1px;
 @cell_left_border_width:        5px;
+
+@navbar-default-bg:               darken(#f8f8f8, 10%);
+@navbar-default-link-color:       darken(#777, 13%);
```

Comment:
Instead of darkening both, why not just darken the text?

Reply:
removed the variable to darken navbar